src/pages/login/tests/login-page.spec.js

```diff
@@ -1,69 +1,91 @@
 import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import { useFormik } from 'formik';
 import { getFromLocalStorage } from '../../../services/local-storage.service';
 import { DARK_THEME, LIGHT_THEME } from '../../../configs/index';
 import Login from '../login';
 import { Loader } from '../../../components/loader/loader';
 
 jest.mock('../login.styles', () => ({ useStyles: () => ({}) }));
 jest.mock('react-redux');
+jest.mock('formik');
 jest.mock('../../../services/local-storage.service');
 
 const dispatch = jest.fn();
-let storage = {
+const mockSubmit = jest.fn();
+const mockChange = jest.fn();
+const mockSetFieldValue = jest.fn();
+const mockBlur = jest.fn();
+const formik = {
+  values: {},
+  handleSubmit: mockSubmit,
+  handleChange: mockChange,
+  touched: { email: true, password: true },
+  errors: {},
+  setFieldValue: mockSetFieldValue,
+  handleBlur: mockBlur
+};
+const storage = {
   loginError: '',
-  userLoading: false
+  userLoading: false,
+  snackBarStatus: false,
+  snackBarSeverity: '',
+  snackBarMessage: '',
+  language: 0
 };
 
-useDispatch.mockImplementation(() => dispatch);
-useSelector.mockImplementation(() => storage);
-
 let theme = LIGHT_THEME;
+let component;
 
 getFromLocalStorage.mockImplementation(() => theme);
 
-const mockSubmit = jest.fn();
-const mockChange = jest.fn();
-const mockSetFieldValue = jest.fn();
-const mockBlur = jest.fn();
-
-jest.mock('formik', () => ({
-  ...jest.requireActual('formik'),
-  useFormik: () => ({
-    values: {},
-    handleSubmit: mockSubmit,
-    handleChange: mockChange,
-    touched: { email: true, password: true },
-    errors: {},
-    setFieldValue: mockSetFieldValue,
-    handleBlur: mockBlur
-  })
-}));
+useDispatch.mockImplementation(() => dispatch);
+useSelector.mockImplementation(() => storage);
+useFormik.mockImplementation(() => formik);
 
 describe('Login page test', () => {
+  beforeEach(() => {
+    component = shallow(<Login />);
+  });
   it('Should render with Light theme', () => {
-    const component = shallow(<Login />);
     expect(component.find('form')).toBeDefined();
   });
-
-  it('Should renfder with Dark theme', () => {
+  it('Should render with Dark theme', () => {
     theme = DARK_THEME;
-    const component = shallow(<Login />);
     expect(component.find('form')).toBeDefined();
   });
 
   it('Should simulate submit event', async () => {
-    const component = shallow(<Login />);
     const input = component.find('[data-cy="staySignedIn"]');
     input.simulate('change', { target: { value: 'Hello' } });
   });
 
   it('Should render Loader', () => {
-    storage = {
-      loginError: '',
-      userLoading: true
-    };
+    storage.userLoading = true;
     const component = shallow(<Login />);
```

Comment:
You have a constant with the "component" name in upper scope, you cant create a new const here.